src/interfaces/useCase.ts

```diff
@@ -2,11 +2,12 @@ import UseCaseResponse from '../classes/UseCaseResponse';
 import TelegramMessage from '../classes/TelegramMessage';
 
 interface UseCase {
-  readonly useCaseName: string;
+  readonly name: string;
+  readonly triggers: string[];
```

Comment:
Okay, auch wenn ich dachte jeder use Case hat **einen einzigen** Trigger.

Reply:
Kannst ja auch ein einzelnes Wort reinmachen, aber ob man bei mir ein "image of the day" oder "photo of the day" will, soll auf das gleiche rauslaufen

Reply:
Ich dachte halt wir machen einzelne Trigger Worts, so das diese in keinem Kontext kommen können
NEWS, FINANCE,...

Reply:
das war mein verständnis

Reply:
Das können wir auch machen, ist mit der jetzigen Lösung ja auch unterstützt. Müssen wir mal in der Gruppe absprechen.